### ISSUE TITLE
Per-draw noise becomes the default regime; the corrected champion becomes the demo model; README, TODO and the paper lead with the corrected result; ADR-0006 supersedes ADR-0003's default

### ISSUE THREAD

**Issue by GovitPowerz** [GovitPowerz/Aerocapture#108]
## Why (portfolio review 2026-09-16, P0 #1, "the most important item")

The repo tells two stories about the same policy. The correction (per-scenario density noise, ADR-0003, paper Appendix E) is disclosed everywhere, but the product still ships the historical regime: the Rust default is `legacy`, the demo flies the shared-path champion and needs `--per-draw` to show the honest number, the README's first table and "One headline" paragraph lead with CVaR99.9 = 123.3, TODO.md calls 123.3 "the deployed sizing headline", and the paper's abstract and conclusion open on 123.3 and reach 163.2 in their last sentence. A reviewer reads that as "documented the caveat, kept the old number". The repo's strongest integrity signal (found a result-changing flaw, quantified it, retrained, re-ran 10^6 scenarios) is buried under its own headline.

Flip it: `per_draw` is the ordinary regime, `legacy` is a reproduction-only mode, the demo flies the corrected champion, and every front-facing number is the corrected one. The 123.3 result survives under a heading that says what it is: the historical result and the correction that replaced it.

## Evidence (line anchors on `main` @ 55628eb)

- `src/rust/src/data/dispersions.rs:495-502`: `NoiseSeeding` derives `Default` with `#[default] Legacy`; doc line 495 says "all committed results and goldens depend on it". `src/rust/src/simulation/run_init.rs:84-92` resolves the absent key through `unwrap_or_default()`. `tests/test_noise_seeding.py:55` pins `test_default_is_legacy`.
- `src/python/aerocapture/demo.py:33-34,51-53`: the demo model is `models/demo/mamba_962` (the frozen champion); `per_draw` is an opt-in flag. `models/demo/mamba_962/README.md` states the model captures 97.9% under per-scenario noise.
- `README.md:13-17` "One headline, one correction" opens on 123.3 and reaches 163.2 in the last sentence; `README.md:26-33` Quick Start comment describes the shared path as the default; `README.md:141-148` the first results table leads with "Deployed | NN - Mamba | 123.3 +- 0.1" with no regime column.
- `TODO.md:75-78`: "the 962-parameter Mamba is the deployed sizing headline -- CVaR99.9 123.3 +- 0.1"; "tag `arxiv-v2`" (the Release is `arxiv-v3`, see #101).
- `articles/paper/paper.typ:82` (abstract leads with 123.3), `:99` (163.2 arrives last), `:1200-1210` (conclusion leads with 123.3), `:1225` (the per-scenario loss stated as a caveat).
- `docs/adr/0003-per-draw-noise-seeding.md`: "the legacy regime stays the default for committed numbers". That decision was right for the arxiv-v3 freeze; it is the thing this issue supersedes.
- The corrected champion exists and is quoted but is not in the tree: `training_output/ou_marginal/ft_mamba_p962/best_model.json` (md5 `1ecc77e492b4f2caa4229b6b69dc2e4b`, untracked), config `configs/training/ou_marginal/ft_mamba_p962.toml` (tracked). Its numbers: `experiments/ou_marginal/RESULTS.md` "Far-tail confirmatory under per-scenario noise": capture 99.9995%, CVaR95 138.7, CVaR99.9 163.0 (s1); three fine-tune seeds 163.2 +- 1.3; raw data `experiments/ou_marginal/confirmatory_marginal.json`.
- Blast radius of flipping the default: the six guidance goldens (`configs/test/*_golden.toml`) have no `[monte_carlo.density_perturbation]` block and use bias navigation, so no noise stream is consumed; `configs/test/test_ekf_mars.toml` does consume one (EKF sensor noise). `test_pyo3_matches_subprocess` compares two paths under the same config, so it is regime-neutral by construction.

## Proposed shape (sketch, not prescriptive)

1. Rust: `#[default] PerDraw` in `dispersions.rs`; the doc comment and `config.rs:1270` say `legacy` is for reproducing pre-2026-08-27 numbers only. Python: `demo.py` flies `per_draw` by default with a `--legacy` opt-in, and prints the regime as it does today. `tests/test_noise_seeding.py::test_default_is_legacy` becomes `test_default_is_per_draw`.
2. Pin `noise_seeding = "legacy"` explicitly in every config whose committed output must not move: all of `configs/test/` (goldens, `test_ekf_mars`, the PyO3 bit-identity configs) and any paper script that re-flies a legacy cell (`articles/paper/scripts/fresh_pool_requote.py`, `confirmatory_eval.py`, `experiments/paper/*.sh` that quote legacy numbers). ADR-0003's consequence "state the regime with every number" is what makes this a pin, not a workaround.
3. Maintainer decision on the name: keep `legacy`, or rename to something unmistakable (`legacy_shared_path`) with the old string hard-erroring (unknown values already do) and every tracked config updated. #105's "one name per key" argues against keeping both.
4. `models/demo/ft_mamba_962/` = copy of `training_output/ou_marginal/ft_mamba_p962/{best_model.json,best_params.json}` + a README with provenance (config, checkpoint lineage from `training_output/mamba_p962_long`, the 2000 per_draw gens, the s1/s2/s3 capture and CVaR99.9 rows, which seed is deployed and why). The demo flies it. The frozen champion stays as `models/demo/mamba_962_legacy/` (the historical row of the demo) or leaves the tree: maintainer call.
5. ADR-0006 "per_draw is the default regime; legacy is reproduction-only", superseding ADR-0003's default clause (0003 status: superseded in part, link both ways). One paragraph on why the default was kept legacy through arxiv-v3.
6. README: "One headline, one correction" opens on the corrected result (capture 99.996% of 10^6, CVaR99.9 163.2 +- 1.3 over three fine-tune seeds, 73 m/s below the best classical scheme and the best dense network) and states the historical number and the correction in one sentence. The first results table becomes the per-scenario table (ft_mamba_962, ft_dense_515, FNPAG, the frozen champion as the historical row) with a regime column (coordinate with #101, which adds the RL row and the regime column to the same table); the 123.3 table moves under "Historical result and evaluation correction". Quick Start comment flips. `TODO.md:75-78` follows.
7. Paper, next arXiv version (coordinate with #101 step 7 and its `CHANGES.md`): abstract and conclusion open on the per-scenario result; the shared-path result and its correction get one sentence up front; Appendix E stays the forensic account. Maintainer decision: which of Sections 6-7 are requoted under per_draw now vs left as the disclosed historical regime (the ou_marginal campaign ran at GA n_pop 60 x 10 sims, not the headline allocation; Appendix E discloses this).

## Constraints (load-bearing; read before touching)

- Numbers must not move: goldens (`tests/reference_data/rust_golden/`), `tests/test_pyo3.py::test_pyo3_matches_subprocess`, `tests/test_run_grid.py`. Pin `legacy` in their configs BEFORE flipping the default and run them; a golden that changes means a config that consumes a noise stream was missed.
- `articles/paper/data/*.json` is untouched; legacy cells are never re-run or re-collected (`collect_runs.py` `LEGACY` block).
- ADR-0002: final eval is report-only; nothing here re-selects a champion. ADR-0003: frozen and per-draw numbers are never compared in one table without a regime column.
- The corrected champion's capture is seed-dependent at the 1e-4 level (5 / 30 / 80 non-captures per 10^6 for s1 / s2 / s3, all genuine crashes); the demo README says which seed is deployed and that deployed policies are confirmatory-screened.
- The feasibility rule (see #109) applies to the deployed cell: `ft_mamba_962` is 0% violations per `RESULTS.md`; state it in the demo README.
- #101 (README results table, `CHANGES.md`, `TODO.md:78`) and #106 (TODO/IMPROVEMENTS roadmap) edit the same files: sequence them or rebase.
- Feature branch `feature/<name>`, never `main`; finish with the `smart-commit` skill (re-syncs CLAUDE.md / README). PyO3 rebuilds from the repo root: `uv run maturin develop --release --manifest-path src/rust/aerocapture-py/Cargo.toml`.

## Acceptance

- [ ] `rg -n -A1 '#\[default\]' src/rust/src/data/dispersions.rs | rg -c PerDraw` -> 1 (rustfmt puts the attribute on its own line); `uv run pytest tests/test_noise_seeding.py -q` passes with a `test_default_is_per_draw` case and no `test_default_is_legacy`.
- [ ] `rg -L 'noise_seeding' configs/test/*.toml` -> empty (every test config pins its regime); `cargo test --release --manifest-path src/rust/Cargo.toml` and `uv run pytest tests/test_pyo3.py tests/test_run_grid.py -q` pass with `git status --porcelain tests/reference_data` empty.
- [ ] `uv run python -m aerocapture.demo` prints a per-scenario regime line and flies `models/demo/ft_mamba_962/best_model.json` (`rg -n 'ft_mamba_962' src/python/aerocapture/demo.py`); `--legacy` reproduces today's numbers on the frozen champion.
- [ ] `md5 -q models/demo/ft_mamba_962/best_model.json` = `1ecc77e492b4f2caa4229b6b69dc2e4b` (or the README names a different deployed seed and its md5).
- [ ] `ls docs/adr/0006-*.md`; `rg -n 'superseded' docs/adr/0003-per-draw-noise-seeding.md` -> 1 hit.
- [ ] `sed -n '1,60p' README.md | rg -n '163\.2'` hits before any `123\.3`; `rg -n '123\.3' README.md TODO.md` hits only where the number is named as the shared-path / historical result (the correction paragraph of "One headline, one correction", the "Historical result" subsection, TODO's program outcome), never as the headline; the first results table has a regime column.
- [ ] Paper: `sed -n '70,105p' articles/paper/paper.typ | rg -n '163\.2'` appears before `123\.3` in the abstract; same for the conclusion block; recorded in `articles/paper/CHANGES.md`. The committed `paper.pdf` is recompiled only when the next-version pass is complete.
- [ ] Gates: `./lint_code.sh`; `./check_all.sh`; `uv run pytest tests -q -m "not slow"`.

## Related

- #101 (RL baseline requote; README results card; `CHANGES.md`): same README table and paper version.
- #106 (presentation; TODO/IMPROVEMENTS roadmap): same TODO.md lines.
- Companion issues from the same review: #109 (the deployed cell must be feasible), #110 (consumes the corrected numbers). Index issue: see #115.
- ADR-0003, `experiments/ou_marginal/RESULTS.md`, paper Appendix E.
